R/packages.R

```diff
@@ -343,8 +356,13 @@ download_wasm_packages <- function(
         meta$path <- glue::glue("packages/{pkg}/{meta$assets[[1]]$filename}")
       }
     }
-    meta
-  })
+
+    cur_metadata[[i]] <- meta
+  }
+
+  if (!is_quiet()) {
+    cli::cli_progress_done()
+  }
```

Comment:
I believe because `cli_progress_bar(.auto_close = TRUE)`, we can remove this code.

```suggestion
```